FreeRTOS/FreeRTOS
# Kernel timers.c tasks.c Unit Test

Kernel timers.c Unit Test

Description
-----------
Kernel timers.c Unit Test with code covverage

Test Steps
-----------
```
timers.c                                        100.0%      100.0 %          251 / 251        100.0 %           26 / 26           100.0 %          77 / 77
```

```

-----------------------
8 Tests 0 Failures 0 Ignored
OK

-----------------------
36 Tests 0 Failures 0 Ignored
```



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
